stle.css

```diff
@@ -0,0 +1,53 @@
+header {
+  background-color: black;
+  color: white;
+  height: 30rem;
+}
+
+.nav-bar {
+  background-color: #363232;
+  height: 3.3em;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 23px;
+  align-items: center;
+}
+
+.header-title {
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  line-height: 1.2;
+  height: 590px;
+  background: black;
+}
+
+.header-title h1 {
+  font-family: "Crete Round", sans-serif;
```

Comment:
- [ ] You have done a great job by using the same fonts that are provided in the Figma design. But the fonts are still not imported which they are not implemented yet. Kindly follow this [link](https://www.w3docs.com/snippets/css/how-to-import-google-fonts-in-css-file.html#:~:text=Open%20Google%20Fonts%20and%20follow,(in%20HTML%20or%20CSS).) to learn how to import fonts from google then import them into your project. 😃